src/translations/fr.translations.json

```diff
@@ -88,7 +88,7 @@
           "description": "Tous nos articles de blog, REX et tutoriels autour du développement Javascript : React.js, Node.js, Nest.js, Next.js, Vue.js, Svelte.js"
         },
```

Comment:
Translations are downloaded from Loco when the app is built, right?
So maybe we don't need to commit these json files?